dns/operationcode.org/records.tf

```diff
@@ -12,9 +12,23 @@ resource "dnsimple_record" "staging_api" {
   value = "${var.k8s-cluster-ingress}"
 }
 
+resource "dnsimple_record" "slack_service" {
+  domain = "${var.hosted-zone}"
+  name = "pybot"
+  type = "CNAME"
+  value = "pyback-lb-197482116.us-east-2.elb.amazonaws.com"
+}
+
+resource "dnsimple_record" "staging_slack_service" {
+  domain = "${var.hosted-zone}"
+  name = "api.staging"
+  type = "CNAME"
+  value = ""
```

Comment:
I'm not sure what the permanent value is for this, waiting to hear back from Allen.